packages/edit-site/src/components/sidebar-global-styles-wrapper/index.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useMemo, useState } from '@wordpress/element';
+import { privateApis as routerPrivateApis } from '@wordpress/router';
+import { layout, seen } from '@wordpress/icons';
+import { useViewportMatch } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import GlobalStylesUI from '../global-styles/ui';
+import Page from '../page';
+import { unlock } from '../../lock-unlock';
+import SidebarButton from '../sidebar-button';
+import StyleBook from '../style-book';
+
+const { useLocation, useHistory } = unlock( routerPrivateApis );
+const GLOBAL_STYLES_PATH_PREFIX = '/wp_global_styles';
+
+export default function GlobalStylesUIWrapper() {
+	const { params } = useLocation();
+	const history = useHistory();
+	const { canvas = 'view' } = params;
+	const [ isStyleBookOpened, setIsStyleBookOpened ] = useState( true );
+	const isMobileViewport = useViewportMatch( 'medium', '<' );
+	const pathWithPrefix = params.path;
+	const [ path, onPathChange ] = useMemo( () => {
+		const processedPath = pathWithPrefix.substring(
+			GLOBAL_STYLES_PATH_PREFIX.length
+		);
+		return [
+			processedPath ? processedPath : '/',
+			( newPath ) => {
+				history.push( {
+					path:
+						! newPath || newPath === '/'
+							? GLOBAL_STYLES_PATH_PREFIX
+							: `${ GLOBAL_STYLES_PATH_PREFIX }${ newPath }`,
+				} );
+			},
+		];
+	}, [ pathWithPrefix, history ] );
+	return (
+		<>
+			<Page
+				actions={
+					<>
+						{ ! isMobileViewport && (
+							<SidebarButton
+								icon={ isStyleBookOpened ? layout : seen }
```

Comment:
Do you think changing icons is a good idea? I found it a bit confusing and I think there are a11y concerns about doing this.

Reply:
Not really. The last commit was just moving them into the global styles panel. I'll try something text-based and folks can pile on with the opinions.